cloudbuild.yaml

```diff
@@ -365,11 +373,10 @@ steps:
           exit 1
         fi
 
-        # TODO: re-add this check once generating embeddings in CI is stable
-        # if [[ $(cat /workspace/rag_prompt_result.txt) != "pass" ]]; then
-        #   echo "rag frontend test failed"
-        #   exit 1
-        # fi
+        if [[ $(cat /workspace/rag_prompt_result.txt) != "pass" ]]; then
+          echo "rag frontend test failed"
```

Comment:
fix in a future PR: this should be:
```
echo "rag prompt test failed"
```

Reply:
Okay, will open a future PR for this fix.